src/.pre-commit-config.yaml

```diff
@@ -1,12 +1,12 @@
 default_language_version:
-  python: python3.7.4
+    python: python3.9
```

Comment:
We could consider to directly bump up to 3.10. I tested it locally, tests did not fail - yet. Let's talk about it in a moment :)